vm/issue_vm.go

```diff
@@ -0,0 +1,63 @@
+package vm
+
+import (
+	"fmt"
+	"os"
+	"bufio"
+	"runtime"
+)
+
+func InitIssueReportVM (dir string, args []string) (*VM, error) {
+	v, err := New(dir, args)
+	v.mode = TestMode
+
+	return v, err
+}
+
+func PrintError(v *VM) {
+	eval := v.mainThread.stack.top().Target
+	err, ok := eval.(*Error)
+	if !ok {
+		fmt.Println("No error detected")
+	}
+	fmt.Printf("# Help! %s\n", err.Type)
+	fmt.Println(err.Message)
+
+
+	fmt.Printf("### Goby version\n%s\n", Version)
+	fmt.Printf("### Go version\n%s\n", runtime.Version())
+	fmt.Printf("### Operating system\n%s\n", runtime.GOOS)
+	fmt.Printf("### GOBY_ROOT\n%s\n", os.Getenv("GOBY_ROOT"))
```

Comment:
Can we also add `GOROOT` and `GOPATH`? Some people install Goby from source, and usually their first issue is related to these two settings.